python/libkvikio/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -21,7 +21,7 @@ function(find_and_configure_nvcomp)
   set(export_args)
   if(KvikIO_EXPORT_NVCOMP)
     # We're vendoring nvcomp and we only want `libnvcomp.so.4`
-    set(export_args BUILD_EXPORT_SET nvcomp)
+    set(export_args BUILD_EXPORT_SET nvcomp INSTALL_EXPORT_SET nvcomp)
```

Comment:
I am planning to help finalize the removal of nvcomp -- I'll take care of removing this in a later PR.